modules/apps/accessibility/accessibility-test/src/testFunctional/macros/AccessibilityMenu.macro

```diff
@@ -26,6 +26,27 @@ definition {
 		}
 	}
 
+	macro deactivateAccessibilityMenu {
+		ApplicationsMenu.gotoPortlet(
+			category = "Configuration",
+			panel = "Control Panel",
+			portlet = ${portlet});
+
+		Click(
+			key_itemName = "Accessibility",
+			locator1 = "ListGroupItem#ITEM_TEXT");
+
+		if (IsElementPresent(checkboxName = "Enable Accessibility Menu", locator1 = "Checkbox#ANY_CHECKBOX_CHECKED")) {
+			Uncheck.uncheckNotVisible(
+				checkboxName = "Enable Accessibility Menu",
+				locator1 = "Checkbox#ANY_CHECKBOX");
+
+			Click(locator1 = "Button#UPDATE");
+
+			Alert.viewSuccessMessage();
```

Comment:
replace with SystemSettings.saveConfiguration

Can we remove the Alert.viewSuccessMessage?

Reply:
I added the `Alert.viewSuccessMessage` because the menu tests were failing before that, AccessibilityMenu was not being activated for some reason. Maybe it was "too fast"... Let's try again, though

Reply:
`Alert.viewSuccessMessage` is already into `SystemSettings.saveConfiguration`